UI/frontend-plugins/decklink-output-ui/DecklinkOutputUI.cpp

```diff
@@ -0,0 +1,67 @@
+#include "DecklinkOutputUI.h"
+#include <obs-module.h>
+#include <util/platform.h>
+#include "decklink-ui-main.h"
+
+DecklinkOutputUI::DecklinkOutputUI(QWidget *parent)
+		: QDialog(parent),
+		  ui(new Ui_Output) {
+	ui->setupUi(this);
+
+	propertiesView = nullptr;
+
+	connect(ui->startOutput, SIGNAL( released() ), this, SLOT(StartOutput()));
+	connect(ui->stopOutput, SIGNAL( released() ), this, SLOT(StopOutput()));
+}
+
+void DecklinkOutputUI::ShowHideDialog()
+{
+	SetupPropertiesView();
+
+	setVisible(!isVisible());
+}
+
+void DecklinkOutputUI::SetupPropertiesView()
+{
+	if (propertiesView)
+		delete propertiesView;
+
+	obs_data_t *settings = obs_data_create();
+
+	OBSData data = load_settings();
+	if (data != nullptr)
+		obs_data_apply(settings, data);
+
+	propertiesView = new OBSPropertiesView(settings,
+						  "decklink_output",
```

Comment:
Style: These lines are indented too far.  Use two additional tabs from the original indent for continued lines.